bindings/java/c/host.c

```diff
@@ -40,31 +40,28 @@ static bool account_exists_fn(struct evmc_host_context* context, const evmc_addr
     const char java_method_signature[] = "(I[B)I";
     assert(context != NULL);
     JNIEnv* jenv = attach();
-    if (jenv != NULL)
```

Comment:
~~If this is removed, then I think you should add an `assert(jenv != NULL)` to all cases it is removed, just like there's an assert for other cases.~~

Actually just add an assert for non-null into `attach` itself.

I'm not sure in what case can `attach` return null?

Reply:
There's no case where attach can return null. I can add an assert for jenv if you like.

Reply:
These asserts are not (should not be) part of the release binary, but helps for finding bugs testing with debug builds.